src/api.test.ts

```diff
@@ -0,0 +1,57 @@
+import { readFile } from "fs/promises";
+import { CURRENT_API_VERSION, ExasolExtension, registerExtension } from "./api";
+
+
+async function readPackageJson() {
+    const fileContentBuffer = await readFile("package.json")
+    const fileContent = fileContentBuffer.toString("utf8")
+    return JSON.parse(fileContent);
+}
+
+async function readPackageJsonVersion() {
+    const packageJson = await readPackageJson()
+    return packageJson.version
+}
+
+describe("api", () => {
```

Comment:
that did just move the issue to a different file, diden't it?
If we forget to update both if them noting will fail.

Reply:
Maybe we could check that the version in the package.json is equal to the highest changes file with the highest version.

Reply:
✅